File: java/problem268.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int e = (n * (n + 1)) / 2;
        int a = 0;

        for (int i : nums) {
            a += i;
        }
        System.out.print(a+" "+e);
        return e - a;
    }
}

File: python/problem268.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums).    #2031ms
        for i in range (n+1):
            if i not in nums:
                return i
            
        n = len(nums).    # 3ms
        e = (n * (n + 1)) / 2
        a = 0

        for i in nums:
            a += i
        
        return e - a
